config/rubocop.yml

```diff
@@ -1,23 +1,33 @@
-inherit_from: .rubocop_todo.yml
+---
 require: rubocop-rspec
 AllCops:
   Include:
   - "**/*.rake"
   - "**/Gemfile"
   - "**/Rakefile"
+  Exclude:
+  - vendor/**/*
+  - db/**/*
   DisplayCopNames: true
+# This ends up being too spammy
 Style/Documentation:
   Enabled: false
-Style/TrailingWhitespace:
-  Enabled: false
-Style/TrailingBlankLines:
-  Enabled: false
 Style/ExtraSpacing:
   Enabled: false
 Style/StringLiterals:
   EnforcedStyle: single_quotes
 Metrics/LineLength:
   Max: 100
+  Exclude:
+    - 'spec/unit/blockscore/candidate_spec.rb'    # temporary
```

Comment:
This is just a personal preference, but I would typically sort lists like this alphabetically so that there's no question where new entries should be added if the list should grow.